shared/utils/filter-machine-rules.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python2
+
+"""
+Takes shorthand file and removes selects not applicable for containers from container profiles.
+"""
+
+
+try:
+    from xml.etree import cElementTree as ElementTree
+except ImportError:
+    import cElementTree as ElementTree
+
+import sys
+
+def parse_xml_file(xmlfile):
+    with open(xmlfile, 'r') as xml_file:
```

Comment:
what is this function for? there is a function in ElementTree that does this and does it way better. `ElementTree.parse`.